src/components/PaymentPlan/CreditCardInformation.jsx

```diff
@@ -20,11 +19,60 @@ import {
 } from '@chakra-ui/react';
 import useHandlePayment from '../../hooks/useHandlePayment';
 import { useUser } from '../../contexts/useUser';
+import SignInPromptBox from '../SignInPromptBox';
+import CustomLargeAlert from '../CustomLargeAlert';
 
 const stripePublicKey = loadStripe(
     'pk_test_51IweHkKvAxvZ5kVeTShMjLwl1ZyDd6u5GtDEMtnWCKcZq3FNj0L0z7ZLmE5Qk6EVaTds84lMbRTfUPj8Aq0Nodt500I8OLMSs4'
 );
 
+const CreditCardInputBox = () => (
+    <Box
+        my={4}
+        p={4}
+        rounded="md"
+        bg={useColorModeValue('brand.200', 'brand.700')}
+    >
+        <CardElement
+            options={
+                useColorMode().colorMode === 'light'
+                    ? {
+                          style: {
+                              base: {
+                                  fontSize: '16px',
+                              },
+                          },
+                      }
+                    : {
+                          style: {
+                              base: {
+                                  fontSize: '16px',
+                                  color: '#F0F0F0',
+                                  '::placeholder': {
+                                      color: '#888888',
```

Comment:
Is it a good idea to hard code these colors?

Reply:
this is under the options json for the stripe Element, we can't pass any variables in as it will turn into a string